network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/BatteryShortCircuitTest.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.network.store.iidm.impl;
+
+import com.powsybl.iidm.network.Battery;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.extensions.BatteryShortCircuit;
+import com.powsybl.iidm.network.extensions.BatteryShortCircuitAdder;
+import com.powsybl.iidm.network.test.BatteryNetworkFactory;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author Franck Lecuyer {@literal <franck.lecuyer@rte-france.com>}
+ */
+class BatteryShortCircuitTest {
```

Comment:
To remove. 
Rename GeneratorShortCircuitTest to ShortCircuitTest and test only the generic class ?